versions.txt

```diff
@@ -112,8 +112,6 @@ proto-google-cloud-redis-v1:0.81.0:0.81.1-SNAPSHOT
 proto-google-cloud-redis-v1beta1:0.81.0:0.81.1-SNAPSHOT
 proto-google-cloud-scheduler-v1:1.21.0:1.21.1-SNAPSHOT
 proto-google-cloud-scheduler-v1beta1:0.81.0:0.81.1-SNAPSHOT
-proto-google-cloud-securitycenter-v1:0.81.0:0.81.1-SNAPSHOT
-proto-google-cloud-securitycenter-v1beta1:0.81.0:0.81.1-SNAPSHOT
```

Comment:
There should also be grpc- and plain google-cloud-securitycenter artifacts here too.